api/inventorypb/agents.proto

```diff
@@ -318,14 +317,12 @@ message ChangeCommonAgentParams {
 // List
 
 message ListAgentsRequest {
-  oneof filter {
-    // Return only Agents started by this pmm-agent.
-    string pmm_agent_id = 1;
-    // Return only Agents that provide insights for that Node.
-    string node_id = 2;
-    // Return only Agents that provide insights for that Service.
-    string service_id = 3;
-  }
+  // Return only Agents started by this pmm-agent.
```

Comment:
let's add comment that only one of them might be used.

Reply:
Done